connectors/sources/mongo.py

```diff
@@ -118,39 +118,29 @@ def get_default_configuration(cls):
                 "label": "Collection",
                 "order": 1,
                 "type": "str",
-                "value": "sample_collection",
-            },
-            "database": {
-                "label": "Database",
-                "order": 2,
-                "type": "str",
-                "value": "sample_database",
+                "value": None,
             },
+            "database": {"label": "Database", "order": 2, "type": "str", "value": None},
             "direct_connection": {
                 "display": "toggle",
                 "label": "Direct connection?",
                 "order": 3,
                 "type": "bool",
-                "value": True,
+                "value": None,
```

Comment:
How is `toggle` displayed if this is `None`? There must be a value for `bool`?  cc @navarone-feekery

Reply:
This is true, `bool` types can't be `None`. They need to be either `True` or `False`. Thanks for spotting that!